docs/client-libraries-java-setup.md

```diff
@@ -42,10 +42,53 @@ If you use Gradle, add the following information to the `build.gradle` file.
 def pulsarVersion = '@pulsar:version@'
 
 dependencies {
-    compile group: 'org.apache.pulsar', name: 'pulsar-client', version: pulsarVersion
+	implementation "org.apache.pulsar:pulsar-client:${pulsarVersion}"
```

Comment:
**[UNRELATED]** updated the configuration to `implementation` and uses the short form of the dependency.